packages/esm-ward-app/src/ward-patient-workspace/ward-patient-action-button.extension.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
It is unclear if or where this is used here.  Is this related to this feature?

Reply:
Things with the `.extension.tsx` suffix are extensions, which are used in the [index file](https://github.com/openmrs/openmrs-esm-patient-management/blob/e82d2c90aa9a28140659bbec69da0afcd130328b/packages/esm-ward-app/src/index.ts#L29). If something has the extension suffix and isn't used in the index file, then something has gone wrong.